packages/fxa-settings/src/components/LinkExpired/index.tsx

```diff
@@ -50,14 +41,18 @@ const getTemplateValues = (linkType: LinkType) => {
       throw new Error('Invalid link type passed into LinkExpired component');
   }
   return templateValues;
-};
+}
 
-const LinkExpired = ({ linkType }: LinkExpiredProps) => {
+const LinkExpired = ({ linkType, resendLinkHandler }: LinkExpiredProps) => {
   // TODO : Metric event(s) for expired link
 
-  const resendLinkHandler = getResendLinkHandler(linkType);
   const templateValues = getTemplateValues(linkType);
-
+  const onClickReceiveNewLink = () => {
+    if (resendLinkHandler == null) {
+      throw new Error('resendLinkHandler missing!')
+    }
+    resendLinkHandler(linkType)
+  }
   return (
     <>
       {/* TODO: Add alertBar for success/failure status of resendLinkHandler */}
```

Comment:
Not related to the edits here, but I notice this comment about alertBar was added in before we created the `Banner` component. That banner component is what we'll need to add below the header to handle success/error messages when we click on resend-link. (This should be a separate ticket)

Reply:
Ah good to know!